src/partials/_contacts.html

```diff
@@ -1,28 +1,31 @@
 <section class="contacts">
   <div class="contacts__background">
-    <h2 class="contacts__section-title">Contacts</h2>
+    <p class="contacts__sectiontitle">Contacts</p>
```

Comment:
tu powinien być nadal nagłówek h2, jest to informacja dla bota, że to nagłówek, inaczej odczyta go on jako paragraf